content/en/blog/2021-12-23-data-curation-baleen.md

```diff
@@ -0,0 +1,57 @@
+---
+title: "Data Curation: A Whale of a Problem"
+slug: "data-curation-baleen"
+date: 2021-12-24T11:37:43-05:00
+draft: false
+image_webp: images/blog/whale.webp
+image: images/blog/whale.jpg
```

Comment:
We prefer to use a different image for each blog post, so I found this one on Flickr Commons and added it to the post (Baleen is a species of whale so I thought this kinda made sense). Please feel free to swap out the image if you like! When you add an image for the blog it needs to be added to the folder called `static/images/blog/` as both a jpg and a webp file. You can use [this website](https://cloudconvert.com/jpg-to-webp) to convert from a jpg to a webp.

Reply:
Will keep this in mind for future posts, thank you.